data/beaconrestapi/src/main/java/tech/pegasys/artemis/beaconrestapi/handlers/beacon/GetValidators.java

```diff
@@ -67,33 +67,34 @@ public GetValidators(final ChainDataProvider chainDataProvider, final JsonProvid
   @OpenApi(
       path = ROUTE,
       method = HttpMethod.GET,
-      summary = "Returns validators that match the specified query.",
+      summary = "Get validators that match the specified query.",
       tags = {TAG_BEACON},
       description =
-          "Returns validator information. If no parameters specified, the first page of validators as of the current epoch are returned.",
+          "Returns validator information.\n\n"
```

Comment:
the actual string line breaks were optional, but i'm guessing this is formatting you actually want, so all good :)

Reply:
Yes, I wanted those breaks :)